test/fonts/test_font_fallback.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+from fpdf import FPDF
+from fpdf.enums import XPos, YPos
+from test.conftest import assert_pdf_equal
+
+HERE = Path(__file__).resolve().parent
+
+
+def test_fallback_font(tmp_path):
+    pdf = FPDF()
+    pdf.add_page()
+    pdf.add_font(family="Roboto", fname=HERE / "Roboto-Regular.ttf")
+    pdf.add_font(family="EmojiOne", fname=HERE / "EmojiOneColor-SVGinOT.ttf")
+    pdf.set_font("Roboto")
+    pdf.set_fallback_fonts(["EmojiOne"])
+    pdf.cell(
+        txt="hello world 😄 😁 😆 😅 ✌ 🤞 🌭 🍔 🍟 🍕",
```

Comment:
```suggestion
        txt="hello **world 😄 😁** 😆 😅 ✌ 🤞 🌭 🍔 🍟 🍕",
```
I suggest using some Markdown markers in the input to check how they are handled